### DISCUSSION TITLE
Responsive Breakpoint prefixes need to be used again for multiple styling

### DISCUSSION THREAD

**Discussion by skyworth01** [tailwindlabs/tailwindcss#12473] [Help]
Currently to give height and width for smaller screens, code would be **sm:h-10 sm:w-10**

But can I give something like **sm:h-10 w-10** and height and width should be applied for smaller screens, I don't know whether there is solution for this or not.

It would be better if multiple styles can be applied by giving only one prefix.

**Comment by wongjn** [comment]
There have been suggestions about adding such a feature. The maintainers have addressed this request before, that you may wish to read up on if you have not already done so:
- [A Twitter thread from the creator of Tailwind, Adam Wathan](https://twitter.com/adamwathan/status/1461519820411789314)
- [A previous GitHub discussion thread with maintainer responses](https://github.com/tailwindlabs/tailwindcss/discussions/8337#discussioncomment-4032611)
